Cargo.toml

```diff
@@ -15,11 +15,10 @@ chacha20 = { version = "0.7", features = ["rng"] }
 clap = { version = "2.33.3", default-features = false }
 data-encoding = "2.3.2"
 delog = "0.1.0-alpha.3"
-generic-array = "0.14.3"
-littlefs2 = "0.2.1"
+generic-array = "0.14.4"
```

Comment:
Just an FYI (in case you didn't know) - version specs "x.y.z" mean "at least this version, at most the latest semver-compatible one". So you don't have to manually change these things (unless you definitely rely on a patch release bugfix), just do `cargo update` and commit `Cargo.lock`.

Reply:
Thanks, yeah, i just wanted to literally align it with the main Trussed repo.